.gitignore

```diff
@@ -1,2 +1 @@
 .htaccess
-README.md
```

Comment:
The readme can be on github :)

Reply:
Wasn't intentional. Naturaly the readme should be on github ;)